.gitlab/bazel/build-deps.yaml

```diff
@@ -8,27 +8,27 @@
 bazel:build-deps:linux-amd64:
   extends: [ .bazel:build-deps, .bazel:runner:linux-amd64 ]
   script:
-    - bazel build //deps/...
+    - bazel build --config=dd-internal //deps/...
```

Comment:
please don't distribute CI-specific options to each and every `bazel` invocation in CI jobs; instead consider favoring the single place of maintenance offered by the `CI config` section in the main `.bazelrc` file, see [below](https://github.com/DataDog/datadog-agent/pull/42969#discussion_r2539158967).